UnitTests/CommonTestUtils/TestAppSettingsAttribute.cs

```diff
@@ -13,6 +13,7 @@ public sealed class TestAppSettingsAttribute : Attribute, ITestAction
         public void BeforeTest(ITest test)
         {
             AppSettings.CheckForUpdates = false;
+            AppSettings.ShowAvailableDiffTools = false;
```

Comment:
Is this needed in other tests too or is TestAppSettingsAttribute used universally?

Reply:
The `TestAppSettingsAttribute` should affect all tests - is included by the `AssemblyInfo.cs` files.